ledger/sandbox/src/test/lib/scala/com/digitalasset/platform/sandbox/ScenarioLoadingITBase.scala

```diff
@@ -159,8 +161,8 @@ abstract class ScenarioLoadingITBase
             val events = responses.flatMap(extractEvents)
             val contractIds = events.map(_.contractId).toSet
 
-            // note how we skip #0, #2 and #5 because of the `pass`es in the scenario.
-            contractIds shouldBe Set("#4:0", "#7:2", "#3:0", "#7:1", "#1:0", "#7:0", "#6:0")
+            // note how we skip #0, #2, #5 and #7 because of the `pass`es in the scenario.
+            contractIds shouldBe Set("#1:0", "#3:0", "#4:0", "#6:0", "#8:0", "#8:1", "#8:2")
```

Comment:
What was the change for the increase of the contract id? A hearbeat after `#6:0`? Wonder how flaky this will be.

Reply:
It's to do with the (somewhat accidental) bug fix, which was pushing out the wrong update to the time. Before, we were not recording the last `pass` in the ledger; now, we are.

In the test scenario, we have:

```daml
  pass (days 10)  -- #0
  party <- getParty "party"
  submit party (create Dummy with operator = party)  -- #1
  pass (seconds 10)  -- #2
  submit party (create DummyWithParam with operator = party)  -- #3
  submit party (create DummyFactory with operator = party)  -- #4
  pass (seconds 10)  -- #5
  submit party (create DummyContractFactory with operator = party)  -- #6
  pass (seconds 10)  -- #7
  other <- getParty "other"
  submitMustFail party (create Dummy with operator = other)
  return ()
```

Contracts `#8:0`–`#8.2` come from `dummyRequest.commands`.

Reply:
Ah sorry. the passing of time is driven by the scenario of course. So no time-based flakiness 👍. Sorry for the noise.

Reply:
No worries; it's a confusing test and it took me quite some time to understand it myself!